src/signals/incident/components/form/MapSelectors/Asset/Selector/Selector.tsx

```diff
@@ -72,14 +71,19 @@ const Selector: FC = () => {
 
   const mapOptions: MapOptions = useMemo(
     () => ({
+      minZoom: 7,
+      maxZoom: 16,
       ...MAP_OPTIONS,
       center,
       dragging: true,
       zoomControl: false,
       scrollWheelZoom: true,
-      minZoom: 7,
-      maxZoom: 16,
-      zoom: coordinates ? MAP_LOCATION_ZOOM : MAP_NO_LOCATION_ZOOM,
+      zoom: coordinates
+        ? Math.min(
+            MAP_LOCATION_ZOOM,
+            MAP_OPTIONS.maxZoom || Number.POSITIVE_INFINITY
+          )
+        : MAP_OPTIONS.zoom,
     }),
```

Comment:
Can you explain why you have dropped the `MAP_NO_LOCATION_ZOOM` const? It would be more readable if the MAP_LOCATION_ZOOM and `MAP_NO_LOCATION_ZOOM` constants are kept and the computation is done over there. Also, I don't see the use of the Math.min part with the Number.POSITIVE_INFINITY. A much simpler and readable solution would be to say ```
const MAP_LOCATION_ZOOM = MAP_OPTIONS.maxZoom ? Math.min(
            MAP_LOCATION_ZOOM,
            MAP_OPTIONS.maxZoom) : MAP_LOCATION_ZOOM 
```

Reply:
And the MAP_OPTIONS_NO_LOCATION_ZOOM is replaced by MAP_OPTIONS.zoom. Are you sure that value can be used? In the amsterdam repo, that value is different than the MAP_NO_LOCATION_ZOOM.

Reply:
We could still put it in a const, that's true.

When there is no location set, you want the map to open at the default zoom level. That's `MAP_OPTIONS.zoom`.

Reply:
I would also like to see a test for this change

Reply:
That's a bit difficult since it's map options passed on to the `Map` component.